docker-compose.yml

```diff
@@ -1,12 +1,27 @@
 version: "3.9"
 services:
   todo_app:
-    image: "todo_app:dev"
+    build:
+      context: .
+      target: development
     ports:
       - "5000:5000"
     env_file:
       - ./.env
     volumes:
       - type: bind
         source: ./todo_app
-        target: /todo_app/todo_app
\ No newline at end of file
+        target: /todo_app/todo_app
+  todo_app_test:
```

Comment:
Nice addition! 😄 
You might have noticed that this doesn't work perfectly (having said that, it might work for you!); sadly watchmedo seems to not pick up file notifications when Docker is backed by WSL, so if you are seeing issues it's not anything you've done wrong!